reverse.sh

```diff
@@ -0,0 +1,12 @@
+count=0
+array=()
+for i in $*
+    do
+    count=$((count+1))
+    array+=($i)
+done
+echo ${array[@]}
+for ((i=$count-1;i>=0;i--))
+  do
+      echo -n "${array[i]} "
+done
```

Comment:
In order to avoid iterating through all the elements twice would it be more efficient to do something like this:

#!/usr/bin/env bash

REVERSED=''

for i in $*
do
        REVERSED=$i' '$REVERSED
done

echo $REVERSED

Reply:
Didn't knew it can be done this way! Thanks for making me aware.

Reply:
I think your idea has a lot of merit. You should resubmit your PR. Maybe the owner of the repo would accept your PR if you made some changes.